bin/format-yarn-outdated.js

```diff
@@ -21,11 +22,15 @@ const cli = meow(`
     --excludes, -e   Path to YAML file which specify package names to exclude
     --changelogs, -c Path to YAML file which specify changelog uris for the packages
 
+  SUBCOMMANDS
+    generate-yaml    Generate YAML file which specify changelog uris for the packages
```

Comment:
> The name is confusing. `excludes.yml` or `changelogs.yml`?

The name `generate-yaml` does not resolve the problem. I think `generate-changelog-yaml` is good.